src/lib/components/Popover/README.mdx

```diff
@@ -0,0 +1,294 @@
+              label="start"
+              onClick={() => setAlign('-start')}
+            />
+            <Button
+              color={align === '' ? 'dark' : 'primary'}
+              label="center"
+              onClick={() => setAlign('')}
+            />
+            <Button
+              color={align === '-end' ? 'dark' : 'primary'}
+              label="end"
+              onClick={() => setAlign('-end')}
+            />
+          </ButtonGroup>
+        </ToolbarItem>
+      </Toolbar>
+      <div
+        style={{
+          display: 'grid',
+          placeContent: 'center',
+          minWidth: '20rem',
+          minHeight: '15rem',
+        }}
+      >
+        <div style={{ position: 'relative' }}>
+          <Placeholder bordered aria-describedby="my-popover-top">
+            Popovers
+            <br />
+            all day long…
+          </Placeholder>
+          <Popover id="my-popover-top" placement={`top${align}`}>
+            Top side
+          </Popover>
+          <Popover id="my-popover-right" placement={`right${align}`}>
+            Right side
+          </Popover>
+          <Popover id="my-popover-bottom" placement={`bottom${align}`}>
+            Bottom side
+          </Popover>
+          <Popover id="my-popover-left" placement={`left${align}`}>
+            Left side
+          </Popover>
+        </div>
+      </div>
+    </>
+  );
+}}
+</Playground>
+
+## Advanced Positioning
+
+While the basic setup can be sufficient in some scenarios, dropping a Popover
+usually won't be so easy. To handle all possible situations and edge cases
+automatically, including smart position updates, we recommend to involve an
+external library designed specifically for this purpose.
+
+ℹ️ The following example is using an external library [Floating UI] (formerly
+Popper.js). To use Floating UI, install it first:
+
+```shell
+npm install --save @floating-ui/react-dom
+```
+
+And import it along with Popover, e.g.:
+
+```js
+import { useFloating } from '@floating-ui/react-dom';
+import { Popover } from '@react-ui-org/react-ui';
+```
+
+As opposed to the [basic setup](#placement), Popover will be placed according to
+its triggering component (`reference`), but still recognizing the closest parent
+element with `position: relative` or `position: absolute` if there is any.
+
+Popover reacts on the `forwardedRef` option, necessary for advanced positioning:
+when `forwardedRef` is set, Popover resets its built-in positioning and relies
+on provided `style`.
+
+👉 Please consult [Floating UI] documentation to understand how it works and to
+get an idea of all possible cases you may need to cover.
+
+<Playground>
+  {() => {
+    const [isPopoverOpen, setIsPopoverOpen] = React.useState(false);
+    const [placement, setPlacement] = React.useState('top');
+    const {
+      x,
+      y,
+      reference,
+      floating,
+      strategy,
+    } = useFloating({ placement });
+    const placementOptions = [
```

Comment:
For the sake of readability I would consider this syntax:
```js
    const placementOptions = [
      'top',
      'top-start',
      'top-end',
      'right',
      'right-start',
      'right-end',
      'bottom',
      'bottom-start',
      'bottom-end',
      'left',
      'left-start',
      'left-end',
    ];
    // All inline styles in this example EXCEPT Popover `style` are for
    // demonstration purposes only.
    return (
      <>
        <SelectField
          label="Placement"
          onChange={e => setPlacement(e.target.value)}
          options={placementOptions.map((el) => ({
            label: el,
            value: el,
          }))}
          value={placement}
        />
```

Reply:
I knew there is a way! 👍🏻